README.md

```diff
@@ -158,19 +159,36 @@ jf 'rt u test-file my-repo –-server-id server-2'
 ```
 
 ### Publishing and accessing the build-info
-[Build-info](https://www.buildinfo.org/) is the metadata of a build. It includes all the details about the build broken down into segments that include version history, artifacts, project modules, dependencies, and everything that was required to create the build. In short, it is a snapshot of the components used to build your application, collected by the build agent.
-See below how you publish the build-info from your pipeline jobs. This section should be placed inside the job after the execution of the JFrog CLI commands used for the build.
+
+[Build-info](https://www.buildinfo.org/) is the metadata of a build. It includes all the details about the build broken
+down into segments that include version history, artifacts, project modules, dependencies, and everything that was
+required to create the build. In short, it is a snapshot of the components used to build your application, collected by
+the build agent.
+See below how you publish the build-info from your pipeline jobs. This section should be placed inside the job after the
+execution of the JFrog CLI commands used for the build.
+
 ```groovy
 stage('Publish build info') {
-  steps {
-    jf 'rt build-publish'
-  }
+    steps {
+        jf 'rt build-publish'
+    }
 }
 ```
-When the job publishes the build-info to Artifactory, you can access it by clicking on the build-info icon, next to the job run.
+
+When the job publishes the build-info to Artifactory, you can access it by clicking on the build-info icon, next to the
+job run.
 
 ![build-info.png](images/readme/build-info.png)
 
+## Using HTTP/S proxy
+
+If you're using a JFrog platform that's situated behind an HTTP/S proxy, you should set up your proxy configuration
+under `Manage Jenkins` > `Manage Plugins` > `Advanced`.
+
+To exclude the JFrog platform from going through a configured proxy, provide your JFrog platform's host details in
+the `No Proxy Host` section. This should be a list of comma-separated hosts.
+Keep in mind that the JFrog CLI is typically downloaded from releases.jfrog.io. You might want to think about adding
+that to your list as well.
```

Comment:
You might want to think about adding that to your list as well.
-->
You may need to add that to your list as well.